### PR TITLE
distributions: add CentOS 10

### PR DESCRIPTION
**Pull request by croissanne**

### CODE REVIEW

File: distributions/centos-10/centos-10.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      "check_gpg": true
+    }]
+  },
+  "aarch64": {
+    "image_types": [ "aws", "guest-image", "image-installer" ],
```
Comment (thozza) [inline distributions/centos-10/centos-10.json:27]: el10 / c10s aarch64 currently supports only:

- qcow2
- openstack
- tar
- wsl
- azure
- ami

**Review by thozza — COMMENTED** [review]
Thanks. I've added two comments.

File: distributions/centos-10/centos-10.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "module_platform_id": "platform:el10",
+  "oscap_name": "cs10",
+  "distribution": {
+    "name": "centos-10",
+    "description": "CentOS Stream 10",
+    "no_package_list": true,
+    "restricted_access": true
+  },
+  "x86_64": {
+    "image_types": [ "ami", "vhd", "aws", "azure", "edge-commit", "edge-installer", "rhel-edge-commit", "rhel-edge-installer", "guest-image", "image-installer", "oci", "vsphere", "vsphere-ova" ],
```
[the remark this answers is not in the document]
Comment (croissanne) [reply distributions/centos-10/centos-10.json]: I added all of them except tar, since that's just the installer payload.

**Review by thozza — APPROVED** [review]
LGTM

**Pull request merged**